remote-api/src/main/java/org/alfresco/repo/webdav/LockMethod.java

```diff
@@ -449,30 +449,18 @@ protected FileInfo createNode(NodeRef parentNodeRef, String name, QName typeQNam
      */
     protected final void createLock(FileInfo lockNode, String userName) throws WebDAVServerException

```

Comment:
It's a refactor with the only logic change being removing 
lockInfo.setTimeoutSeconds(getLockTimeout());
to pass the getLockTimeout() explicitly in 
getDAVLockService().lock(lockNode.getNodeRef(), lockInfo, getLockTimeout());
instead of passing timeout through date operations